movie-mlops-project/serving/streamlit_app.py

```
@@ -0,0 +1,473 @@
+                st.write(f"**개봉:** {release_year}")
+            
+            if genres:
+                genres_str = ', '.join(genres[:3])  # 최대 3개 장르
+                st.write(f"**장르:** {genres_str}")
+        
+        with col2:
+            # 포스터 이미지 (130px로 복원)
+            if poster_url:
+                try:
+                    st.image(poster_url, width=130)
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)